promgen/templates/promgen/rule_detail.html

```diff
@@ -28,6 +28,10 @@ <h1>Rule: {{ rule.name }}</h1>
 <div class="panel panel-primary">
     <div class="panel-heading">{{rule.name}}</div>
     <div class="panel-body">
+        <div>
+            <label>Description:</label>
+            <p>{{rule.description}}</p>
+        </div>
         <pre v-pre>{{rule|rule_dict|pretty_yaml}}</pre>
     </div>
```

Comment:
Description is currently not a required field, so perhaps we should do this as a conditional .

```suggestion
{% if rule.description %}
    <div class="panel-body">
        <div>
            <label>Description:</label>
            <p>{{rule.description}}</p>
        </div>
        <pre v-pre>{{rule|rule_dict|pretty_yaml}}</pre>
    </div>
{% endif %}
```

So that we don't have just a label if there is no custom description.